internal/projects/creator_test.go

```diff
@@ -107,3 +128,16 @@ func TestProvisionSelfEnrolledProjectInvalidName(t *testing.T) {
 	}
 
 }
+
+// prepareTestToken creates a JWT token with the specified roles and returns the context with the token.
+func prepareTestToken(t *testing.T, roles []any) context.Context {
```

Comment:
This should probably take an existing context, so you can use it as a wrapper.  (Just a standard pattern, like `context.WithValue()`)